src/GovUk.Education.ExploreEducationStatistics.Content.Api/appsettings.Development.json

```diff
@@ -1,7 +1,6 @@
 {
   "enableSwagger": true,
   "PublicStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://data-storage:10000/devstoreaccount1;QueueEndpoint=http://data-storage:10001/devstoreaccount1;",
-  "PublisherStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;QueueEndpoint=http://data-storage:10001/devstoreaccount1;",
```

Comment:
Yep good for this to go!🙂

We don't need to remove it from the App Service config in the Azure environments because it doesn't exist. It would be around https://github.com/dfe-analytical-services/explore-education-statistics/blob/dev/infrastructure/templates/template.json#L1560 in the Arm template config if it was set.

It's only in `appsettings.Development.json` so that locally the Content API could queue a message to Publisher to regenerate all content on startup. This didn't happen in any other environment so Content API didn't need this connection string.

Reply:
Cool understood!